fontbe/src/paths.rs

```diff
@@ -57,6 +57,11 @@ impl Paths {
         self.glyph_dir.join(safe_filename(name, ".gvar"))
     }
 
+    fn kern_slice_file(&self, segment: usize) -> PathBuf {
```

Comment:
segment, slice, fragment?... so many synonyms

Reply:
I'll try to land on fragment :)